app/RouteHandlers/PayPalNotificationHandler.php

```diff
@@ -61,13 +61,13 @@ public function handle( Request $request ): Response {
 					->setPaymentType( $post->get( 'payment_type', '' ) )
 			);
 
-			return new Response( '', 200 );
+			return new Response( '', Response::HTTP_OK ); # PayPal expects an empty response
 		} catch ( PayPalPaymentNotificationVerifierException $e ) {
 			// TODO: log error
 			// TODO: let PayPal resend IPN?
+			// TODO: is this the right Response?
```

Comment:
Not sure. PayPal will resend the IPN over and over again if the returned status code is something other than 200. And it will be failing over and over again, because the data that is verified will never change. The old application is doing this, though.

I think, we should send back 200, but notify the fundraising team for further investigation.